.circleci/config.yml

```diff
@@ -416,6 +416,16 @@ jobs:
     resource_class: xlarge
     steps:
       - checkout
+      - run:
+         name: Login into gcr
+         command: |
+            echo ${GCLOUD_SERVICE_KEY} | docker login -u _json_key --password-stdin https://gcr.io
+
+      - run:
+         name: Build docker builder image
+         command: |
+                docker build -t test --target builder .
```

Comment:
Are we sure we want this to run before the clean? 
To be honest, this cache should be automatically used, I don't see why we need to explicitly build this target.

Also, this uses a bad tag `test`

Reply:
As I saw docker removes build intermediate containers. So I thought builder image was just removed during full build. 
I will move it a little bit down after clean and will fix the tag. I changed it just for local test but forgot to change it back